packages/mobile/src/screens/update-required-screen/NewVersionPrompt.tsx

```diff
@@ -0,0 +1,65 @@
+import { View } from 'react-native'
+
+import IconDownload from 'app/assets/images/iconDownload.svg'
+import { Button, GradientText, Text } from 'app/components/core'
+import { makeStyles } from 'app/styles'
+
+const useStyles = makeStyles(({ palette, spacing, typography }) => ({
+  contentContainer: {
+    paddingTop: spacing(32),
+    paddingBottom: spacing(8),
+    paddingHorizontal: spacing(6)
+  },
+  header: {
+    fontSize: 24,
```

Comment:
Typography value for this?

Reply:
tru

Reply:
actually this is bigger than even h1? leaving as is